src/app/api/navigationManager/route.ts

```diff
@@ -0,0 +1,32 @@
+// app/api/navigationManager/route.ts
+import { NextResponse } from "next/server";
+import { client } from "studio/lib/client";
+import { token } from "studio/lib/token";
+
+// Ensure the client is configured with the token
```

Comment:
I think I want to expand on this comment, maybe add something to readme.

In order to fetch in custom sanity component we need the token, however the newest setup of the env ensures the token is only accessible server-side — but all sanity components run client side. This means that all calls inside sanity has to be created in the api-folder as their own routes.

Reply:
We're going to have to do the same with most of the custom Sanity components. I'll give it some thought to see if there's a better approach to this.